### PR TITLE
Redis rdb

### PR DESCRIPTION
**Pull request by bharel**

### CODE REVIEW

File: yellowbox/extras/redis.py [hunk 1]
```diff
@@ -1,22 +1,29 @@
...
 
 
 class RedisService(SingleContainerService, RunnableWithContext):
-    def __init__(self, docker_client: DockerClient, image='redis:latest', **kwargs):
-        super().__init__(
-            create_and_pull(docker_client, image, publish_all_ports=True, detach=True),
-            **kwargs
-        )
+    def __init__(self, docker_client: DockerClient, image='redis:latest',
+                 redis_file: bytes = None, **kwargs):
```
Comment (bentheiii) [inline yellowbox/extras/redis.py:18]: why is `redis_file` not IO?

File: yellowbox/extras/redis.py [hunk 2]
```diff
@@ -1,22 +1,29 @@
...
+        container = create_and_pull(docker_client, image, publish_all_ports=True, detach=True)
+        self.started = False
```
Comment (bentheiii) [inline yellowbox/extras/redis.py:20]: maybe this should be standard in all containerservices?

File: yellowbox/extras/redis.py (hunk 1 above)
Comment (bentheiii) [reply yellowbox/extras/redis.py:18]: also `Optional`

File: yellowbox/extras/redis.py (hunk 2 above)
Comment (bharel) [reply yellowbox/extras/redis.py:34]: dunno 🤷

**Review by bentheiii — APPROVED** [review]

**Pull request merged**